security/tpm/samples/AuthenticationProviderTpmSimulator/AuthenticationProviderTpmSimulator.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netstandard2.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;netstandard2.0</TargetFrameworks>
```

Comment:
next up to remove net5.0?

Reply:
That should be doable, maybe also remove the tests for net5.0 target next